fastlane_core/lib/fastlane_core/update_checker/changelog.rb

```diff
@@ -21,6 +21,11 @@ def show_changes(gem_name, current_version)
       end
 
       def releases(gem_name)
+        # We have to follow redirects, since some repos were moved
+        # away into a separate org
+        # Taken from https://github.com/excon/excon/issues/115#issuecomment-40647211
+        Excon.defaults[:middlewares] << Excon::Middleware::RedirectFollower
```

Comment:
Sure, if the change can be made to just apply to a single request, that would also solve the immediate concern 👍